docs/organization-units.md

```diff
@@ -49,3 +49,109 @@ to describe what it contains (e.g. "sierra-leone-districts.geojson").
+```json
+{
+  "organisationUnitLevels": [
+    {
+      "name": "Chiefdom",
+      "level": 3,
+      "id": "tTUf91fCytl"
+    },
+    {
+      "name": "District",
+      "level": 2,
+      "id": "wjP19dkFeIk"
+    },
+    {
+      "name": "Facility",
+      "level": 4,
+      "id": "m9lBJogzE95"
+    },
+    {
+      "name": "National",
+      "level": 1,
+      "id": "H1KlN4QIauv"
+    }
+  ]
+}
+```
+
+Once we have the level information, we can use that to fetch geometries. See a sample below.
+
+**GET api call:** ```api/organisationUnits.geojson?level=1&level=2&level=3&level=4```
+
+**sample output** (for brevity lots of coordinates are truncated from the second feature data)
+```json
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "id": "plnHVbJR6p4",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          -12.9487,
+          9.0131
+        ]
+      },
+      "properties": {
+        "code": "OU_211234",
+        "name": "Ahamadyya Mission Cl",
+        "level": "4",
+        "parent": "QywkxFudXrC",
+        "parentGraph": "ImspTQPwCqd/PMa2VCrupOd/QywkxFudXrC",
+        "groups": [
+          "GGghZsfu7qV",
+          "oRVt7g429ZO"
+        ]
+      }
+    },
+    {
+      "type": "Feature",
+      "id": "YuQRtpLP10I",
+      "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+          [
+            [
+              -11.3516,
+              8.0819
+            ],
+            [
+              -11.3553,
+              8.0796
+            ],
+            ...
+            ,
+            [
+              -11.3516,
+              8.0819
+            ]
+          ]
+        ]
+      },
+      "properties": {
+        "code": "OU_539",
+        "name": "Badjia",
+        "level": "3",
+        "parent": "O6uvpzGd5pu",
+        "parentGraph": "ImspTQPwCqd/O6uvpzGd5pu",
+        "groups": [
+          "gzcv65VyaGq"
+        ]
+      }
+    }
+  ]
+}
+```
+The output contains geometry information of all orgunits living on the mentioned levels. Two types of geometry information are returned - a point (with lat,lng) or a polygon (with several points of lat,lng forming the shape). As can be seen from the output, apart from the gemoetry information additional detailed properties - for example code, name, level, parent, parent graph and groups - about each orgunit are returned.
+
+Result can be saved as a .geojson file and latter imported to a DHIS2 instance via Import/Export app 
```

Comment:
I would suggest that we remove this paragraph (and screenshot) as it is less relevant to import org unit data that was already exported from DHIS2. Instead we could build on the branch from @karimbahgat: https://github.com/dhis2/climate-tools/tree/karim